ipykernel/ipkernel.py

```diff
@@ -110,11 +113,9 @@ def __init__(self, **kwargs):
 
         self.executing_blocking_code_in_main_shell = False
 
-        from .debugger import Debugger, _is_debugpy_available
```

Comment:
It looks like https://github.com/ipython/ipykernel/pull/1223 intentionally introduced this lazy import. Let's make sure we're addressing the need there as well before making this an eager import at the top of the file.

Reply:
In particular, does this change mean that `python -m ipykernel install --sys-prefix` has spurious warnings?

Reply:
CC @krassowski who made this lazy import chnage earlier this year.

Reply:
Thanks for spotting, yes this is intentional to address https://github.com/ipython/ipykernel/issues/1198